src/test/java/org/sonarlint/intellij/finding/issue/IssueFixtures.kt

```diff
@@ -54,6 +59,19 @@ fun aCoreIssue(file: PsiFile, textRange: TextRange? = TextRange(0, 0, 0, 1)) = o
     override fun getType() = RuleType.BUG
     override fun getRuleKey() = "ruleKey"
     override fun flows() = mutableListOf<Flow>()
+    override fun getCleanCodeAttribute() = CleanCodeAttribute.defaultCleanCodeAttribute()
+
+    override fun getImpacts(): MutableMap<SoftwareQuality, ImpactSeverity> {
+        return Map.of(
```

Comment:
Also, use kotlin map and not java map

`        return mutableMapOf(
            SoftwareQuality.MAINTAINABILITY to ImpactSeverity.HIGH,
            SoftwareQuality.RELIABILITY to ImpactSeverity.MEDIUM,
            SoftwareQuality.SECURITY to ImpactSeverity.LOW
        )`